src/core/server/base.js

```diff
@@ -179,19 +192,38 @@ function baseServer(routes, createStore, { appInstanceName = appName } = {}) {
             </I18nProvider>
           );
 
+          const asyncConnectLoadState = store.getState().reduxAsyncConnect.loadState || {};
+          const apiErrors = [];
+          Object.keys(asyncConnectLoadState).forEach((item) => {
```

Comment:
I'd be okay with this mapping and filtering, slightly less efficient but reads more cleanly to me:

```js
const apiErrors = Object.keys(asyncConnectLoadState[item].error)
  .map((item) => asyncConnectLoadState[item].error)
  .filter((item) => item);